### DISCUSSION TITLE
AZQR Generated Xlsx File Cost Tab

### DISCUSSION THREAD

**Discussion by riaz-faisal-apl** [Azure/azqr#275] [Q&A] [answered]
azqr scan generates an excel file with multiple tabs but the cost Tab shows the accumulated cost since the creation of that subscription. Is it possible to get scan results for a particular month or last month or some specific period. 
Command executed.
```azqr scan```
cost tab output 
| 2024-09-01 | 2024-12-04 | xxxxxxxx-xxxx-xxxx-xxxx-xxxxx | xxxxxxxx-xxxx-xxxx-xxxx-xxxxx | Azure Container Apps |
|------------|------------|-------------------------------|-------------------------------|----------------------|

**Comment by cmendible** [comment, marked as answer]
Hi @riaz-faisal-apl, the cost should be showing the costs for the subscription for the last 3 months:

https://github.com/Azure/azqr/blob/14c6c9e2a2a27d60eecbe0e3fb8afbfdca28294e/internal/scanners/cost.go#L49

We could add a parameter to set the number of months, between 1 and 12 being 3 the default, to look back.

**Reply by riaz-faisal-apl** [reply]
Thanks
